components/SearchComponents/OptionsBar/OptionsBar.module.css

```
@@ -60,7 +59,7 @@
 .options {
   display: flex;
 
-  @media (min-width: $smallRem) {
+  @media (min-width: 40em) {
   }
```

Stylelint (17.13.0)

[error] Empty block (block-no-empty)

(block-no-empty)